src/app/_components/BadgeCardGrid.tsx

```diff
@@ -0,0 +1,22 @@
+import clsx from 'clsx'
+
+type GridColumnConfig = 'smThree' | 'smTwoLgFour'
+
+type BadgeCardGridProps = {
+  cols: GridColumnConfig
+  children: React.ReactNode
+}
+
+const baseGridStyles = 'grid grid-cols-1 gap-6 sm:auto-rows-fr'
+const extendedGridStyles: Record<GridColumnConfig, string> = {
+  smThree: 'sm:grid-cols-3',
+  smTwoLgFour: 'sm:grid-cols-2 lg:grid-cols-4',
+}
+
+export function BadgeCardGrid({ cols, children }: BadgeCardGridProps) {
+  return (
+    <ul className={clsx(baseGridStyles, extendedGridStyles[cols])}>
+      {children}
+    </ul>
+  )
+}
```

Comment:
Wondering if it would make sense to move the `BadgeCardGrid` component in the same file as `CardWithBadge`? Since it's specifically made for `CardWithBadge` layout, and I assume we're not gonna use it in other cases? Just an thought.

Reply:
Same [>>](https://github.com/FilecoinFoundationWeb/filecoin-foundation/pull/273/files#r1664258302) 🙏🏼

Reply:
Let's keep these concerns separate for now!